components/Hero.tsx

```diff
@@ -29,13 +29,13 @@ export default function Hero() {
         </p>
         <div className="grid grid-cols-1 justify-center pt-5 pb-2 sm:pb-8 md:grid-cols-2 md:space-x-5 md:pt-8">
           <div className="flex items-center justify-center md:justify-end">
-            <Button classes="text-xl w-full" href={routes.chaptersUrl}>
+            <Button classes="text-xl !w-64" href={routes.chaptersUrl}>
```

Comment:
we still need it to be full width on mobile, is there some error causing us to need the important tag?
``` suggestion
            <Button classes="text-xl w-full md:w-64" href={routes.chaptersUrl}>
```